.env.dev

```diff
@@ -0,0 +1,6 @@
+LLAMA_STACK_BASE_URL=http://localhost:5001
+LLAMA_STACK_CLIENT_LOG=debug
+LLAMA_STACK_PORT=5001
+LLAMA_STACK_DIR=.
```

Comment:
I don't know if "." is a good value honestly at all

Reply:
Removed